packages/@internationalized/date/src/calendars/IslamicCalendar.ts

```diff
@@ -79,6 +86,13 @@ export class IslamicCivilCalendar implements Calendar {
   }
 }
 
+/**
```

Comment:
i'm unclear the difference between tabular and civil

Reply:
The only difference is the epoch. Civil uses Friday, tabular uses Thursday

Reply:
doh, it was one day of difference, sorry, I just missed it